internal/provider/resource_instance_test.go

```diff
@@ -277,12 +308,10 @@ resource "oxide_instance" "{{.BlockName}}" {
 				Config: initialConfig,
 				Check:  checkResourceInstanceIP(resourceName, instanceName),
 			},
-			// Detach the external IP.
```

Comment:
It'd be good to still test the case where we remove the `external_ips` block entirely like it was before in addition to the new test. Let's also leave these comments. They make the tests clearer for people new to the codebase.